tests/test_symbolics.py

```diff
@@ -562,6 +562,31 @@ def test_minmax():
     assert np.all(f.data == 4)
 
 
+@pytest.mark.parametrize('dtype,expected', [
+    (np.float32, ("fmaxf", "fminf")),
+    (np.float64, ("fmax", "fmin")),
+])
+def test_minmax_precision(dtype, expected):
+    grid = Grid(shape=(5, 5), dtype=dtype)
+
+    f = Function(name="f", grid=grid)
+    g = Function(name="g", grid=grid)
+
+    eqn = Eq(f, Min(g, 4.0) + Max(g, 2.0))
```

Comment:
Would use numpy numbers to check dtype is correct

Reply:
wdym exactly? I can think of at least two ways...

I landed on the above because that's the natural way people write the equations and it doesn't matter whether it's 4, 4.0, or 4.0F, because in the end, `g` will drive the `fmin/fmax` generation

Reply:
@mloubout I'm gonna merge this because I don't like broken CI, but as soon as you elaborate on how you want this test improved, I'll mkae the change and push it together with one of the upcoming branches (at least one more to go in before christmas)